Cargo.toml

```diff
@@ -34,10 +34,11 @@ strip = true
 [dependencies]
 deltachat_derive = { path = "./deltachat_derive" }
 deltachat-time = { path = "./deltachat-time" }
+deltachat-contact-utils = { path = "./deltachat-contact-utils" }
```

Comment:
Maybe just `contact-utils = ...` like for `ratelimit`?

Reply:
The latter is a crate called `ratelimit` stored in the directory `deltachat-ratelimit`. Since `deltachat-contact-utils` are utils specific for deltachat, so I don't think the crate should just be called `contact-utils`.